packages/components/src/FormatFile/FormatFile.css

```diff
@@ -51,12 +51,61 @@
   margin: 0 auto;
 }
 
-.progress {
+.fileContentWrapper {
+  display: flex;
+  width: inherit;
+  height: inherit;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.thumbailIconWrapper {
+  align-self: center;
+}
+
+.thumbnailFilename {
   display: flex;
   position: absolute;
-  top: 0;
+  bottom: var(--space-small);
+  left: var(--space-small);
+}
+
+.thumbnailFilenameSmall {
+  display: flex;
+  position: absolute;
+  bottom: var(--space-smaller);
+  left: var(--space-smaller);
+}
+
+.thumbnailNonImage::before {
+  content: "";
+  display: block;
+  position: absolute;
   right: 0;
   bottom: 0;
+  z-index: 1;
+  width: 20px;
```

Comment:
This width and height should be 16px or `--space-base`

Reply:
quick note on this - we've had to conditionally apply this to account for the difference in width & height for the large thumbnail. 
We have matched the figma to be 16px/`--space-base` on the default thumbnail however with larger font-size and increased margin in the larger thumbnail I needed to apply `--space-large` to cover the file name completely, otherwise we see the below:

![Screen Shot 2022-05-27 at 11 54 42 AM](https://user-images.githubusercontent.com/104797704/170735226-3d4ec17e-6129-4b34-b46e-6b6d04fc49fb.png)